test/malli/generator_test.cljc

```diff
@@ -61,6 +61,25 @@
       (is (test-presence special? {:gen/infinite? true
                                    :gen/NaN? true}))
       (is (not (test-presence special? nil)))))
+  
+  (testing "float properties"
+    (let [infinity? #(or (= % ##Inf)
+                         (= % ##-Inf))
+          NaN? (fn [x]
+                 (#?(:clj  (fn [v]
+                             (or (infinity? v) (Float/isNaN v)))
+                     :cljs js/isNaN)
+                  x))
+          special? #(or (NaN? %)
+                        (infinity? %))
+          test-presence (fn [f options]
+                          (some f (mg/sample [:float options]
+                                             {:size 1000})))]
+      (is (test-presence infinity? {:gen/infinite? true}))
+      (is (test-presence NaN? {:gen/NaN? true}))
+      (is (test-presence special? {:gen/infinite? true
+                                   :gen/NaN? true}))
+      (is (not (test-presence special? nil)))))
```

Comment:
It was a bit tricky writing this test since I want to use `Float/isNaN`, but sometimes when generating Infinity the tests would error because of the value being too big to coerce into a float.

Note, I don't think this test generates `Float` instances, so I think it should be updated to check for that but I'm unsure how to do that. If someone know how please let me know 🙏